plugin/src/main/java/com/h3xstream/findsecbugs/taintanalysis/TaintFrameModelingVisitor.java

```diff
@@ -190,6 +190,8 @@ public void visitSIPUSH(SIPUSH obj) {
 
     @Override
     public void visitGETSTATIC(GETSTATIC obj) {
+        // Scala uses some classes to represent null instances of objects
+        // If we find one of them, we will handle it as a Java Null
         if (obj.getLoadClassType(getCPG()).getSignature().equals("Lscala/collection/immutable/Nil$;")) {
```

Comment:
Ok. Perfect.
If this add to much complexity to TaintFrameModelVisitor, we will have to reconsider. At the moment, this one condition is reasonable and I don't see side effect.